### PR TITLE
fix: types for latest AWX version

### PR DESCRIPTION
**Pull request by oleghalin**
Latest AWX version Host object incompatible types fixed. 

- [Docs reference](https://ansible.readthedocs.io/projects/awx/en/latest/rest_api/api_ref.html)

Error during collecting facts:

```
│ Error: Unable to fetch Host
│ 
│   with module.rke-node1.awx_host.vm_host,
│   on modules/proxmox-cloudinit/main.tf line 107, in resource "awx_host" "vm_host":
│  107: resource "awx_host" "vm_host" {
│ 
│ Unable to load Host with id 96: got json: cannot unmarshal number into Go struct field Host.last_job of type awx.Job
```

### CODE REVIEW

**Comment by josh-silvas** [comment]
Thank you for the contribution @oleghalin! Can you please update the [VERSION](https://github.com/josh-silvas/terraform-provider-awx/blob/main/VERSION) file so this drafts a new release on merge? Thanks!

**Comment by Aldetux** [comment]
Since this issue is still open, can I mention that this is not just for host an issue at the moment. The same is true for Users:

 ```
 Unable to create Username got json: cannot unmarshal string into Go struct
 field User.type of type int
 ```
 
 In docs i see for User:
 `type: Data type for this user. (choice)` 
 
 If nobody has time to fix this in the near future, I might give it a shot, but this would be a first time doing something in go, so for now, I tick to reporting it ;)

**Comment by oleghalin** [comment]
@Aldetux thanks, i just included fix to this MR. 

@josh-silvas i bumped version

**Review by josh-silvas — APPROVED** [review]

**Comment by josh-silvas** [comment]
Thanks for the contributions! This is released as 1.3.1 (https://registry.terraform.io/providers/josh-silvas/awx/1.3.1)

File: tools/goawx/types.go [hunk 1]
```diff
@@ -688,7 +688,7 @@ type JobEvent struct {
 // User represents an user.
 type User struct {
 	ID              int         `json:"id"`
-	Type            int         `json:"type"`
+	Type            string      `json:"string"`
```
Comment (aeber) [inline tools/goawx/types.go:691]: Hi @oleghalin ! I think this should be changed back to `json:"type"` as the field in the API is called "type" not "string", right?

**Pull request merged**